Nodejs/Product/Nodejs/SharedProject/CommonProjectNode.DiskMerger.cs

```diff
@@ -22,41 +25,56 @@ internal abstract partial class CommonProjectNode
         /// </summary>
         private sealed class DiskMerger
         {
-            private readonly Stack<DirState> remainingDirs = new Stack<DirState>();
+            private readonly ConcurrentStack<(string Name, HierarchyNode Parent)> remainingDirs = new ConcurrentStack<(string, HierarchyNode)>();
```

Comment:
>ConcurrentStack [](start = 29, length = 15)

Does this still need to be a `ConcurrentStack`?

Reply:
yeah, continueMerge could be on any thread, where the worker is on the UI thread.